### PR TITLE
Redux changes for SchedulingPage

### PR DESCRIPTION
**Pull request by firejake308**
This is a smaller PR that isolates the changes made to Redux in order to support the new SchedulingPage

### CODE REVIEW

**Comment by firejake308** [comment]
The order for merging is this one, then #196, then #188

**Comment by gannonprudhomme** [comment]
While ik it's a relatively straightforward code, I think we should at least have a basic test for each of the actions just for thoroughness

**Comment by firejake308** [comment]
Tests have been added

File: autoscheduler/frontend/src/redux/actions/schedules.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import {
+  AddScheduleAction, ADD_SCHEDULE, RemoveScheduleAction, REMOVE_SCHEDULE,
+  ReplaceSchedulesAction, REPLACE_SCHEDULES,
+} from '../reducers/schedules';
+import Meeting from '../../types/Meeting';
+
```
Comment (gannonprudhomme) [inline autoscheduler/frontend/src/redux/actions/schedules.ts:6]: Unnecessary extra empty line

**Comment by gannonprudhomme** [comment]
Probably should've just done request changes since you'll have to re-request my reviews anyways

**Comment by firejake308** [comment]
Sorry to do this again, but I had to merge one of the files

**Pull request merged**